src/devices/Hcsr501/samples/README.md

```diff
@@ -0,0 +1,60 @@
+# Example of HC-SR501
+
+## Hardware Required
+* PIR Motion Sensor - HC-SR501
+* LED
+* 220 Ω resistor
+* Male/Female Jumper Wires
+
+## Circuit
+![](circuit_bb.png)
+
+HC-SR501
+* VCC - 5V
+* GND - GND
+* OUT - GPIO 17
+
+LED
+* VCC & 220 Ω resistor - GPIO 27
+* GND - GND
+
+## Code
+* First, initialize LED.
+    ```C#
+    GpioController ledController = new GpioController(PinNumberingScheme.Logical);
+    // open PIN 27 for led
+    ledController.OpenPin(27, PinMode.Output);
+    ```
+
+* Second, you need to create a HC-SR501 object.
+    ```C#
+    using(Hcsr501 sensor = Hcsr501(17, PinNumberingScheme.Logical))
+    {
+
+    }
+    ```
+
+* Finially, in the loop, use property IsMotionDetected to detect motion.
+    ```C#
+    while (true)
+    {
+        if (sensor.IsMotionDetected == true)
```

Comment:
1. I think complete sample is slightly clearer instead of splitting this into "here is how you create object", "here is how you detect".

Will IsMotionDetected be 1 and stay 1 for some time when movement is detected? If so some comment on that might be useful (if it allows changing the timing it might be worth adding to the sample)